web/app.js

```diff
@@ -1330,6 +1330,78 @@ var PDFViewerApplication = {
       });
     }
   },
+
+  unbindEvents: function pdfViewBindEvents() {
+    var eventBus = this.eventBus;
```

Comment:
Please use `let` instead of `var` when adding new code (applies elsewhere in the patch as well).